style/styles.css

```diff
@@ -42,7 +44,61 @@ header {
   color: #6070ff;
 }
 
-.menu-items {
+/*
+.menu{
+  display: none;
+}
+*/
+
+.menu {
+  position: fixed;
+  transform: translateY(-100%);
+  transition: transform 0.3s;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  background: #6070ff;
+  list-style: none;
+  opacity: 0.9;
+  padding-top: 2rem;
+}
+
+.menuItem {
+  display: block;
+  margin: 2rem 4rem;
+  font-size: 1.8rem;
+  color: white;
+  text-decoration: none;
+}
+
+.menuItem:hover {
+  text-decoration: underline;
+}
+
+.showMenu {
+  transform: translateY(0);
+}
+
+.hamburger {
+  display: block;
+  position: fixed;
+  z-index: 100;
+  top: 1rem;
+  right: 1rem;
+  padding: 4px;
+  border: #fff solid 1px;
+  background: #6070ff;
+  cursor: pointer;
+}
```

Comment:
- [ ] Great job, As I can see here your humburger menu is not the Figma template so kindly remove the border and the backgorund to match the Figma design as much as possible ✅✅

- Your design
![image](https://user-images.githubusercontent.com/58553711/179864350-9519dd15-e43c-48b0-b58b-11b0ba73c37a.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/179864576-f2b6a19a-2ee8-4c9b-b9bc-11781ca8bc52.png)